service/credential_adapter/credential_adapter.go

```diff
@@ -131,32 +131,18 @@ func (c *credentialAdapterClient) IssueVerifiableCredential(cred_type string, di
 	return VerifiableCredential{Doc: doc, Alias: CreateDefaultAlias(), Type: cred_type}, nil
 }
 
-func (c *credentialAdapterClient) PresentVerifiableCredential(creds []model.SerializedDocument, proof model.Proof, did string, nonce string, signature []byte) error {
+func (c *credentialAdapterClient) PresentVerifiableCredential(vp map[string]interface{}, did string, nonce string, signature []byte) error {
```

Comment:
So that we could use this type here: `func (c *credentialAdapterClient) PresentVerifiableCredential(vp VPType, did string, nonce string, signature []byte) error {`

Reply:
I think we can investigate that in a follow up, this was just a minimal change for now